lib/Swagger/src/Discussion/Api/ContributionApi.php

```diff
@@ -11,7 +11,9 @@
  */
 
 /**
- * 
```

Comment:
The good news is that the rest of this PR is just updating the generated swagger code due to Discussion API changes.